semeio/jobs/scripts/gendata_rft.py

```diff
@@ -173,14 +173,18 @@ def main_entry_point():
     options = arg_parser.parse_args()
     logger.setLevel(options.log_level)
 
-    well_names = [w_info[0] for w_info in options.well_and_time_file]
-
-    trajectories = {
-        wname: Trajectory.load_from_file(
-            filepath=os.path.join(options.trajectory_path, wname + ".txt")
-        )
-        for wname in well_names
-    }
+    context_errors = []
+    trajectories = {}
+    for well_name, *_ in options.well_and_time_file:
```

Comment:
Better! I was wondering if I should suggest `well_name, *_` instead of the `wellinfo[0]`, just reads better. Thanks for changing